### PR TITLE
fixed find_packages

### PR DESCRIPTION
**Pull request by mserranom**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def version():
         return "0." + os.getenv('CIRCLE_BUILD_NUM')
 
 
-# emit a VERSION file the CLI can use to check current version
+# emit a VERSION file the CLI can use to cpheck current version
```
Comment (prashastk) [inline setup.py]: Typo

File: setup.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def version():
     author="productml.com",
     author_email="info@productml.com",
     url="https://github.com/productml/blurr",
-    packages=find_packages(include="blurr"),
+    packages=find_packages(),
     data_files=[
```
Comment (prashastk) [inline setup.py]: Instead of this ```include_package_data=True``` should work too.

Comment (mserranom) [reply setup.py]: where?

Comment (prashastk) [reply setup.py]: Replace data_files=[] with include_package_data=True

Comment (mserranom) [reply setup.py]: ah

**Pull request merged**